### PR TITLE
Dev tools doc

### PR DESCRIPTION
**Pull request by 0golovatyi**
Update documentation for how to use `set_credentials`

### CODE REVIEW

File: docs/tabpy-tools.md [hunk 1]
```diff
@@ -30,6 +31,26 @@ The URL and port are where the Tableau-Python-Server process has been started -
 more info can be found in the
 [server section](server-startup.md#Command-Line-Arguments) of the documentation.
 
+## Authentication
+
+When TabPy is configured with authentication feature on client code
+has to specify credential to use during model deployment with
```
Comment (johng42) [inline docs/tabpy-tools.md]: This should be

When TabPy is configured with the authentication feature on, client code
has to specify the credentials to use during model deployment with

File: CHANGELOG [hunk 2]
```diff
@@ -8,6 +8,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Improvements
 
 - Added basic access authentication (all methods except /info)
+- tabpy-tools can deploy to TabPy with auth on
```
Comment (johng42) [inline CHANGELOG]: should this be

tabpy-tools can deploy models to TabPy with authentication on

File: docs/tabpy-tools.md (hunk 1 above)
Comment (0golovatyi) [reply docs/tabpy-tools.md]: fixed

File: CHANGELOG (hunk 2 above)
Comment (0golovatyi) [reply CHANGELOG]: fixed

**Pull request merged**